src/pipeline.cc

```diff
@@ -693,10 +694,15 @@ class PipelineWorker : public AsyncWorker {
           // the overlayGravity was used for extract_area, therefore set it back to its default value of 0
           baton->overlayGravity = 0;
         }
-        // Ensure overlay is premultiplied sRGB
-        overlayImage = overlayImage.colourspace(VIPS_INTERPRETATION_sRGB).premultiply();
-        // Composite images with given gravity
-        image = Composite(overlayImage, image, baton->overlayGravity);
+        if(baton->overlayCutout) {
+          // 'cut out' the image, premultiplication is not required
```

Comment:
What will be the best option? Will there be any difference in the image if we don't pre-multiply it?